spec/controllers/review_mapping_controller_spec.rb

```diff
@@ -523,11 +523,11 @@
           allow_any_instance_of(ReviewMappingController).to receive(:automatic_review_mapping_strategy).with(any_args).and_return(true)
           params = {
             id: 1,
-            max_team_size: 1,
```

Comment:
Was there any specific cause for these renamings?

Reply:
@namanshrimali , To make them more readable and it was mentioned in the documentation as well